pkgs/leak_tracker/lib/src/leak_tracking/primitives/_test_helper_detector.dart

```diff
@@ -27,8 +27,16 @@ final _startFrame = RegExp(
   r')',
 );
 
-bool isCreatedByTestHelper(String trace, List<RegExp> exceptions) {
-  final frames = trace.split('\n');
+/// If true, leaking objects created by test helpers will be ignored.
```

Comment:
super-nit: this should document this method, not necessarily where it is used.

/// Returns whether the leak reported by [objectCreationTrace] was created by a test helper.
///
/// See also [LeakTesting.createdByTestHelpers].